Packs/MandiantAdvantageThreatIntelligence/Integrations/MandiantAdvantageThreatIntelligence/MandiantAdvantageThreatIntelligence.py

```diff
@@ -1048,8 +1049,22 @@ def get_new_indicators(
         new_indicators_list = list(
             filter(last_updated_filter(start_date), new_indicators_list)  # type: ignore
         )
+        return new_indicators_list
+    else:
+        updated_indicators = []
+        # For Indicators of Compromise only
+        for indicator in new_indicators_list:
+            # Check if indicator should be added no matter what
+            # E.g. it meets the `param` requirements
+            if indicator.get("mscore") >= minimum_mscore:
+                updated_indicators.append(indicator)
+            else:
+                existing_indicators = list(IndicatorsSearcher(value=indicator.get("value")))
+                if len(existing_indicators) > 0 and int(existing_indicators[0].get("total")) > 0:
```

Comment:
Same note here. Is `total` an optional key? Because at its current state, if it's missing, it will try to convert `None` to an `int`, resulting in an unexpected error of:
`TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'`

If you expect this key to always be there, please use:
```suggestion
                if len(existing_indicators) > 0 and int(existing_indicators[0]["total"]) > 0:
```

If not, handle cases where it's missing.